pretix_eth/exporter.py

```diff
@@ -57,7 +56,7 @@ def refund_to_row(refund):
     fiat_amount = refund.amount
     token_amount = refund.info_data.get("amount", "")
 
-    wallet_address = WalletAddress.objects.filter(order_payment=refund.payment).first()
+    wallet_address = None  # todo WalletAddress.objects.filter(order_payment=refund.payment).first()
```

Comment:
iWhat is the point of this and the next line if we are setting it to None?

Reply:
Refunds are disabled under single receiver node for private key management reasons.